redash/query_runner/drill.py

```diff
@@ -0,0 +1,239 @@
+import os
+import logging
+import requests
+import re
+
+from dateutil import parser
+
+from redash.query_runner import BaseQueryRunner, register
+from redash.query_runner import TYPE_STRING, TYPE_DATETIME, TYPE_INTEGER, TYPE_FLOAT, TYPE_BOOLEAN
+from redash.utils import json_dumps, json_loads
+
+logger = logging.getLogger(__name__)
+
+
+# Drill returns request result as strings, so we have to guess the actual column type
+def guess_type(string_value):
```

Comment:
We have the same or almost the same code in the google_spreadsheets and query_results query runners. Could you please move the code into `redash.query_runner.guess_type` and use that instead?